test/ginkgo-ext/scopes.go

```diff
@@ -453,10 +455,13 @@ func wrapNilContextFunc(fn func(string, func()) bool) func(string, func()) bool
 // execute AfterAll. This is tracked via scope.focusedTests and .normalTests.
 // This function is similar to wrapMeasureFunc.
 func wrapItFunc(fn func(string, interface{}, ...float64) bool, focused bool) func(string, interface{}, ...float64) bool {
-	if !countersInitialized {
-		countersInitialized = true
+	if rootScope.counter == -1 {
```

Comment:
Shouldn't these also be protected by mutex?

Reply:
I don't think it's necessary, and the other mutex is not strictly necessary either, but as I stated in the commit message - it's to highlight the brittle nature of this hacky code. So I'll put the counter behind a method.